src/containers/upcoming-events/ducks/thunks.ts

```diff
@@ -1,11 +1,18 @@
-import { EventInformation, EventsThunkAction } from './types';
+import { getPrivilegeLevel } from '../../../auth/ducks/selectors';
+import { PrivilegeLevel } from '../../../auth/ducks/types';
 import { upcomingEvents } from './actions';
+import { EventInformation, EventsThunkAction } from './types';
 
 export const getUpcomingEvents = (): EventsThunkAction<void> => {
-  return (dispatch, getState, { publicApiClient }) => {
+  return (dispatch, getState, { publicApiClient, protectedApiClient }) => {
+    const eventsFetcher: () => Promise<EventInformation[]> =
+      getPrivilegeLevel(getState().authenticationState.tokens) ===
```

Comment:
⛏️ Not sure why we cant return this value directly and avoid defining a variable here.